services/jobs/src/main/java/com/linkedin/openhouse/jobs/services/JobsRegistry.java

```diff
@@ -51,6 +54,15 @@ public JobLaunchConf createLaunchConf(String jobId, JobConf conf) {
     return builder.proxyUser(conf.getProxyUser()).args(extendedArgs).build();
   }
 
+  private void populateSparkProps(
+      Map<String, String> executionConf, Map<String, String> sparkPropsMap) {
+    String memory = executionConf.getOrDefault("memory", null);
+    if (!Strings.isNullOrEmpty(memory)) {
+      sparkPropsMap.put("spark.driver.memory", memory);
+      sparkPropsMap.put("spark.executor.memory", memory);
```

Comment:
Should we allow different values for driver vs executor? I believe for most OOMs, it's the driver memory we need to bump.

Reply:
I believe both SE and OFD perform materialized ops to collect orphaned files and snapshots to expire which requires them to have more driver memory, but perform large distributed jobs to identify the orphaned files as well for example. I didn't have a reason to keep executor less than driver and wanted to keep it simple for usage. Hence choose same param for driver and executor.
I'm okay to separate them out if you prefer though. LMK.

Reply:
I think it is OK to separate driver and executor, given we have top-level `ExecutionConf`

Reply:
We can start with the current version